### PR TITLE
IPv6 support

### PR DESCRIPTION
**Pull request by kaaax0815**
IPv6 Support thanks to @crazyracer98, @pheiduck, @peterlewis 
Close: #138 
Close: #781 
Original PR: #781 

As I didnt want to deal with merge conflicts, I manually reapplied the code and updated it. I included the contributers to the original pr as co-authors

### CODE REVIEW

**Comment by pheiduck** [comment]
@kaaax0815 Thank you for the rework, additionally please fix lint errors:
`npm run lint -- --fix`

**Comment by kaaax0815** [comment]
@pheiduck Should work now

File: src/lib/WireGuard.js [hunk 1]
```diff
@@ -71,6 +75,10 @@ module.exports = class WireGuard {
         // await Util.exec('iptables -A INPUT -p udp -m udp --dport 51820 -j ACCEPT');
         // await Util.exec('iptables -A FORWARD -i wg0 -j ACCEPT');
         // await Util.exec('iptables -A FORWARD -o wg0 -j ACCEPT');
+        // await Util.exec(`ip6tables -t nat -A POSTROUTING -s ${WG_DEFAULT_ADDRESS6.replace('x', '')}/120 -o eth0 -j MASQUERADE`);
```
Comment (PratikMahajan) [inline src/lib/WireGuard.js:78]: here `eth0` should be replaced with `WG_DEVICE`
https://github.com/wg-easy/wg-easy/pull/1103/files#diff-65b7aac2a6e96abf37ec0c4c0c24a4e94ac70fc6a0702d74f45f79b900716f79R74

File: src/config.js [hunk 2]
```diff
@@ -14,18 +15,36 @@ module.exports.WG_CONFIG_PORT = process.env.WG_CONFIG_PORT || process.env.WG_POR
...
-iptables -A FORWARD -o wg0 -j ACCEPT;
-`.split('\n').join(' ');
+// Set WG_POST_UP to allow IPv6 NAT and forwarding only if the required kernel module is available
+const modules = childProcess.execSync('lsmod', {
+  shell: 'bash',
+});
+module.exports.WG_POST_UP = process.env.WG_POST_UP;
+if (!process.env.WG_POST_UP) {
+  module.exports.WG_POST_UP = `  
+  iptables -t nat -A POSTROUTING -s ${module.exports.WG_DEFAULT_ADDRESS.replace('x', '0')}/24 -o eth0 -j MASQUERADE;
```
Comment (PratikMahajan) [inline src/config.js:34]: interface is being hardcoded here, it should be `-o ${module.exports.WG_DEVICE}`
Similarly, need to fix it for ipv6

**Pull request merged**